src/utils.ts

```diff
@@ -45,6 +45,58 @@ export const getNotesFromPath = async (
   });
 };
 
+export async function getTagsFromNote(
+  file: TFile,
+  vault: Vault
+): Promise<string[]> {
+  const fileContent = await vault.cachedRead(file);
+  const frontMatter = parseYaml(fileContent.split('---')?.[1] ?? '') || {};
```

Comment:
```
YAMLParseError: Unexpected scalar token in YAML stream: "Even though it sounds appealing enough
```

I mean yeah. It just doesn't work like this.
I think for now let's stub it with if startswith `---` and then figure out if there are any additional bugs sprinkled on top of this buggy code